index.html

```diff
@@ -73,124 +87,158 @@ <h2 class="proj-title">Tonic</h2>
                 <span class="intro-1">CANOPY</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images/mobile/Counter.png" alt="dot">
                 <span class="intro-2">Back end Dev</span>
-                <img class="dot" src="images/Counter.png" alt="dot">
+                <img class="dot" src="images//mobile/Counter.png" alt="dot">
                 <span class="intro-3">2015</span>
               </div>
               <p class="p-descrip">A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <p class="p-descrip4">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                
               </p>
               <ul class="proj-lan">
                 <li class="lan">HTML</li>
+                <li class="lan">Ruby on Rails</li>
                 <li class="lan"> CSS</li>
                 <li class="lan">Javascript</li>
               </ul>
               <div class="btn-div">
                 <button class="btn" type="submit"> See Project</button>
               </div>
             </div>
+            </div>
           </div>
         </section>
-        <section class="sec-3">
-          <div class="section3-div">
+          <section>
+             <div class="section-3">
             <div class="third-section">
-              <div>
                 <h1 class="title">
                   About Myself
                 </h1>
-                <p class="intro">
+                <p class="introb">
                   I'm a software developer! I can help you build a product , feature or website Look through some of my
                   work and experience! If you like what you see and have a project you need coded, don’t hesitate to
                   contact me.
                 </p>
                 <h3 class="getintouch">
                   LET'S CONNECT
                 </h3>
-              </div>
               <ul class="social-media-icon">
-                <li><a class="social-media" href="#"><img src="images/github.png" alt="github"></a></li>
-                <li> <a class="social-media" href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
-                <li><a class="social-media" href="#"><img src="images/anglelist.png" alt="anglelist"></a> </li>
-                <li><a class="social-media" href="#"><img src="images/twitter.png" alt="twitter"></a></li>
-                <li><a class="social-media" href="#"><img src="images/medium.png" alt="medium"></a></li>
+                <li><a class="social-media" href="#"><img src="images/mobile/github.png" alt="github"></a></li>
+                <li> <a class="social-media" href="#"><img src="/images/mobile/linkedin.png" alt="linkedin"></a> </li>
+                <li><a class="social-media" href="#"><img src="images/mobile/anglelist.png" alt="anglelist"></a> </li>
```

Comment:
- [ ] Please also make sure the About myself section is properly aligned and is  similar to the figma design as much as possible, currently there are not properly aligned, and also missing the gap between the paragraph, let;s connect, social icons and the button, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230735336-65572f64-4ebf-4191-b1c2-f982bdf87fb9.png)

figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230735362-5dac884a-811a-48ea-aff3-35fb413489cd.png)

Reply:
Please consider taking a look at this suggestion, since it hasn't been addressed yet:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/230744218-b6c04fb0-c6e2-4846-b10b-a13d10410669.png) | ![image](https://user-images.githubusercontent.com/77069011/230744207-dfc07750-c932-4063-8e8c-6c4f283aabfa.png) |

- [ ] If you look closely, there's a space between the paragraph and the `Let's Connect` title, and also between the social media icons and the button. Please apply this spacing since it will give your About section a better look and feel for future people browsing your portfolio! 🤓 🔧 

- [ ] Also, you aligned the title and paragraph but not the button, please consider aligning it! 🤓 🔧